Mega Man/Game.cs

```diff
@@ -113,14 +113,14 @@ private void LoadFile(string path, List<string> pathArgs = null)
 
             BasePath = project.BaseDir;
 
-            PixelsDown = project.ScreenHeight;
-            PixelsAcross = project.ScreenWidth;
-
-            if (ScreenSizeChanged != null)
-            {
-                ScreenSizeChangedEventArgs args = new ScreenSizeChangedEventArgs(PixelsAcross, PixelsDown);
-                ScreenSizeChanged(this, args);
-            }
+            //PixelsDown = project.ScreenHeight;

```

Comment:
These properties are used by the engine for a bunch of things. Do a find references to see. I think the solution to this problem should be in MainForm Game_ScreenSizeChanged.